.zenodo.json

```diff
@@ -0,0 +1,62 @@
+{
+    "description": "<p>Climate Model Output Rewriter</p>",
+    "access_right": "open",
+    "license": "BSD-3-Clause",
+    "title": "PCMDI/CMOR: CMOR v3.7.0",
+    "version": "v3.7.0",
+    "upload_type": "software",
+    "publication_date": "2022-08-02",
+    "creators": [
+        {
+            "name": "Mauzey, Chris",
+            "affiliation": "Lawrence Livermore National Laboratory"
+        },
+        {
+            "name": "Durack, Paul",
+            "affiliation": "Lawrence Livermore National Laboratory"
```

Comment:
@mauzey1 are ORCID identifiers allowed? If yes, mine is 0000-0003-2835-1438, Karl's is 0000-0002-6491-2135, Piotr's is 0000-0002-1849-7620, @dnadeau4's is 0000-0002-9622-3289 (correct?), @ehogan's is 0000-0002-1514-5224 (correct?), Zeb is 0000-0002-4767-2723, @jonseddon is 0000-0003-1302-1049 (correct?) and @wachsylon is 0000-0003-4389-6911 (correct?). @doutriaux1, @jkettleb, @TobiasWeigel, @kjoti, @jmrgonza, @EdwardBetts, if you folks would like your ORCID included, can you ping this thread with the correct identifier please?

Reply:
I've added the ORCIDs you provided as well as my own.

Reply:
@mauzey1 perfect. It's very early in Aust, and late in the UK, so can we park this for ~18+ hrs awaiting feedback if we get it and then proceed with the release then?

Reply:
@durack1 I think next Monday would be an ideal release day.

Reply:
Hi @durack1, I don't knowingly have an ORCID (I didn't create the one you linked to, so it may belong to someone else); may I request that the ORCID for me is removed, please? :)

Reply:
@durack1

I'm removing the ORCID that was given to @ehogan, and adding the ORCID for Zeb.  How much longer should we wait for people's responses on whether or not they want their ORCID in the Zenodo author list?  Should we just remove the ones that we aren't certain on?